src/pe_reports/report_generator.py

```diff
@@ -264,13 +259,15 @@ def main():
     if not os.path.exists(validated_args["OUTPUT_DIRECTORY"]):
         os.mkdir(validated_args["OUTPUT_DIRECTORY"])
 
+    try:
+        soc_med = validated_args["--soc_med_included"]
+    except Exception as e:
+        LOGGER.info(f"Social media should not included: {e}")
+        soc_med = False
```

Comment:
If you follow my earlier suggestion related to docopt, you can remove this block and just make the change below.
```suggestion
```